build/lib/glossarpy/ __init__.py

```diff
@@ -0,0 +1 @@
+__all__ = ["GlossTxt", "GlossEntry", "GreatGloss"]
```

Comment:
`build/` should probably be in your `.gitignore` file and not in your main code base.